contracts/interchain-token-service/tests/deploy_remote_interchain_token.rs

```diff
@@ -0,0 +1,112 @@
+mod utils;
+
+use axelar_soroban_std::assert_contract_err;
+use axelar_soroban_std::events;
+use interchain_token_service::error::ContractError;
+use interchain_token_service::event::InterchainTokenDeploymentStartedEvent;
+use soroban_sdk::{testutils::Address as _, Address, BytesN, Env, IntoVal, String};
+use soroban_token_sdk::metadata::TokenMetadata;
+use utils::setup_env;
+use utils::setup_gas_token;
+
+fn setup_token_metadata(env: &Env, name: &str, symbol: &str, decimal: u32) -> TokenMetadata {
```

Comment:
not a very useful helper, use TokenMetaData directly

Reply:
Or implement a TokenMetaDataExt trait with a `new` method in `utils/mod.rs` for a nicer usage